pages/cloudflare/caching.mdx

```diff
@@ -183,36 +183,17 @@ export default defineCloudflareConfig({
 
 #### SSG site
 
-If your site is static, you do not need a Queue nor a Tag Cache. You can also use the `long-lived` mode of the R2 Incremental Cache to make reading from the store faster.
+If your site is static, you do not need a Queue nor a Tag Cache. You can use a read-only Workers Static Assets-based incremental cache for the prerendered routes.
 
-<Tabs items={["wrangler.jsonc", "open-next.config.ts"]}>
-<Tabs.Tab>
-
-```jsonc
-{
-  "name": "<WORKER_NAME>",
-  // ...
-
-  // R2 incremental cache
-  "r2_buckets": [
-    {
-      "binding": "NEXT_INC_CACHE_R2_BUCKET",
-      "bucket_name": "<BUCKET_NAME>",
-    },
-  ],
-}
-```
-
-</Tabs.Tab>
+<Tabs items={["open-next.config.ts"]}>
```

Comment:
Keep `wrangler.jsonc` with the asset binding?

Reply:
We don't actually have the asset binding referenced anywhere in this doc since it's part of the getting started with open next steps and is a requirement for every project